### ISSUE TITLE
postprocessing interface smooth transition

### ISSUE THREAD

**Issue by nikohansen** [numbbo/coco#359]
I would like the post-processing call _from any location_ to be as simple as 
```
  python -m cocopp folder [more_folders]
```
The first thing to do is to change all testing calls like 
```
  python bbob_pproc
```
to 
```python bbob_pproc -t``` or ```python bbob_pproc --test```
At the moment this changes nothing, but later the call
```python -m cocopp``` should preferably show the help.

**Comment by nikohansen** [comment]
For now `'', 'all', '-t', '--t'` invokes testing, otherwise `rungeneric.py` is called. This behavior will (presumably) change soon.

**Comment by nikohansen** [comment]
Currently the post-processing can be installed and called from anywhere via `python -m bbob_pproc folder_name [more_folder_names]`.

**Comment by brockho** [comment]
In order to have only one interface whether one calls the postprocessing from the command line or from the IPython shell, we should consider the following (after today's discussion with @nikohansen):

```
python -c "import cocopp; cocopp.main(...)"
```
and for testing
```
python -c "import cocopp; cocopp.test(...)"
```

in which the arguments of `cocopp.main` should allow to specify algorithm data folders (as (i) a single string, separated by blanks, as (ii) a list of strings, or as (iii) separate string arguments) together with at least a parameter `setup_file='my_file.py'` with which it is possible to change settings, e.g. in `genericsettings.py`. How the latter is optimally set-up is not so clear yet. One possibility to explore is the following:

```
vars = ast.literal_eval(open("my_file.py"))
for k,v in vars.__dict__.items():
   cocopp.genericsettings.__dict__[k] = v
```

which should work for small changes. Another option is to import the file instead of evaluating literally:
```
import my_file
for k,v in my_file.__dict__.items():
   cocopp.genericsettings.__dict__[k] = v
```

or by adding `from my_file import *` in the end of `genericsettings.py`. All setting changes should be also supported via a `cocopp.set_settings(filename)` functionality.

**Comment by brockho** [comment]
Can be closed with #1152 as a follow-up.
